README.md

```diff
@@ -18,7 +17,25 @@
         def application do
           [applications: [:phoenix_pubsub]]
         end
-
+        
+        
+## Initialisation (without Phoenix)
+
+
+        defmodule MyApp do
```

Comment:
There are too many spaces here. The indentation on the installation section is due to being in a list. 4 spaces here should be sufficient.